tests/conftest.py

```diff
@@ -147,3 +150,49 @@ def cmdline_test(request):
     # Import qutebrowser.app so all cmdutils.register decorators get run.
     import qutebrowser.app  # pylint: disable=unused-variable
     return request.param
+
+
+class ConfigStub:
+
+    """Stub for the config module.
+
+    Attributes:
+        data: The config data to return.
+    """
+
+    def __init__(self, signal):
+        """Constructor.
+
+        Args:
+            signal: The signal to use for self.changed.
+        """
+        self.data = {}
+        self.changed = signal
+
+    def section(self, name):
+        """Get a section from the config.
+
+        Args:
+            name: The section name to get.
+
+        Return:
```

Comment:
I think this should be ``Returns:``, right?

http://sphinx-doc.org/latest/ext/example_google.html